pkg/apis/eventing/v1alpha1/broker_lifecycle_test.go

```diff
@@ -13,7 +13,6 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
```

Comment:
nit: unintended?

Reply:
Yes. In vscode editor the line above test package is deceiving